patches/chrome-browser-permissions-permission_uma_util.cc.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/chrome/browser/permissions/permission_uma_util.cc b/chrome/browser/permissions/permission_uma_util.cc
+index 14efe3ccb3906516ba5c13a1c1ba78fb2ccd2d7d..1f24a413c7d023ad9b0d921d3b277b50e1132d8b 100644
+--- a/chrome/browser/permissions/permission_uma_util.cc
++++ b/chrome/browser/permissions/permission_uma_util.cc
+@@ -94,6 +94,8 @@ std::string GetPermissionRequestString(PermissionRequestType type) {
+       return "SecurityKeyAttestation";
+     case PermissionRequestType::PERMISSION_PAYMENT_HANDLER:
+       return "PaymentHandler";
++    case PermissionRequestType::PERMISSION_AUTOPLAY:
++      return "Autoplay";
```

Comment:
`NOTREACHED()` after this will CHECK failed and crash browser if we don't handle this case.
And I prefer putting it into enum because it can save us from something that we don't handle in compile time when upstream add other usage somewhere.

Reply:
fixed in 462c5d292b71d86a93fbeada516942d1142fc7ad